src/components/chip-set/chip-set.e2e.ts

```diff
@@ -3,6 +3,7 @@ import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
 describe('limel-chip-set', () => {
```

Comment:
> I'd also rename this commit `style: fix lint errors` to `fixup! feat: add label to chip-set of types other than input` :) just to keep things cleaner 

I even think all of the commits can be squashed into one, because they all belong to the same functional change: enabling labels.

So I'll rename the commits to fixups.

Reply:
⚡